### PR TITLE
Createpost

### PR DESCRIPTION
**Pull request by SadiqZak**
Feats
- Create Post
- Add/Edit Comment
- Like/Dislike Post

### CODE REVIEW

**Comment by SadiqZak** [comment]
Please login using the pre existing credentials by navigating to the LOGIN page using the login button on the top right corner to inspect the preview and test the features

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_JWT_SECRET = mozak
```
Comment (Devansu-Yadav) [inline .env:1]: Please hide env variables using .gitignore file

File: src/Components/Header/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  return (
+    <div className="header">
+      <div className="header-wrapper">
+        <Logo />
+        <div className="search">
+          <input placeholder="Search" type="text" />
+          <span className="search-icon material-icons">search</span>
+        </div>
+        <div className="header_right">
+          {/* { !auth.isAuthenticated && (
```
Comment (Devansu-Yadav) [inline src/Components/Header/Header.jsx:18]: Commented code can be refactored and deleted

**Review by Devansu-Yadav — APPROVED** [review]
- UI Looks clean
- CSS is well written
Looks good to merge

**Pull request merged**